test/sharness/t0054-dag-car-import-export-data/README.md

```diff
@@ -0,0 +1,24 @@
+# Dataset description/sources
```

Comment:
@Stebalien forgot to ask you:
```
du -sh test/sharness/t0054-dag-car-import-export-data/
2.9M	test/sharness/t0054-dag-car-import-export-data/
```

Is this acceptable to be included in the repo as-is? If not - I can rework the test to use xz ( and skip if not found )

Reply:
I'd compress them. It's not terrible, but 2.9MiB is a bit much.

Reply:
Fix incoming with the comments ( no code changes )